### PR TITLE
Add MesloLGS font config steps for Alacritty

### PR DESCRIPTION
**Pull request by scribbles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -556,24 +556,7 @@ applications on your system. Configure your terminal to use this font:
...
-    italic:
-      family: MesloLGS NF
-      style: Italic
-    bold_italic:
-      family: MesloLGS NF
-      style: Bold Italic
-
-    size: 11.0
-  ```
+- **Alacritty**: If you have not already generated a config file, copy `alacritty.yaml` from [Alacritty releases](https://github.com/alacritty/alacritty/releases) to `~/.config/alacritty/`. Change `family` to `MesloLGS NF` by following [these](https://github.com/alacritty/alacritty/wiki/Changing-the-default-font) instructions.
```
Comment (romkatv) [inline README.md:559]: The instructions must specify how to set Alacritty font to MesloLGS NF. This PR isn't quite there.

The first sentence seems confusing (why generated? why copy?) and inaccurate (Alacritty looks for the config in 4 places and the suggested file is neither the first nor the last of them). The second sentence links to a wiki that shows how to set font to "Droid Sans Mono Slashed for Powerline".

Comment (scribbles) [reply README.md]: It is definitely confusing. The multiple locations Alacritty reads from are based on the XDG base directory spec but the project's own wiki entry for font config only mentions one specific directory. $XDG_CONFIG_HOME being equal to ~/.config/ in the spec. d369295 simplifies much more.

File: README.md [hunk 2]
```diff
@@ -556,7 +556,23 @@ applications on your system. Configure your terminal to use this font:
...
+      style: Regular
+    bold:
+      family: "MesloLGS NF"
+      style: Bold
+    italic:
+      family: "MesloLGS NF"
+      style: Italic
+    bold_italic:
+      family: "MesloLGS NF"
+      style: Bold Italic
```
Comment (z0rc) [inline README.md:573]: AFAIK you don't need to specify font for every style. Just `normal` block should be sufficient, other styles will properly fall back to normal font family.

File: README.md [hunk 3]
```diff
@@ -556,6 +556,23 @@ applications on your system. Configure your terminal to use this font:
 - **Guake**: Right Click on an open terminal and open *Preferences*. Under *Appearance* 
   tab, uncheck *Use the system fixed width font* (if not already) and select `MesloLGS NF Regular`. 
   Exit the Preferences dialog by clicking *Close*.  
+- **Alacritty**: Edit or create `~/.config/alacritty/alacritty.yml`. Append the `font` section below:
+  ```yaml
+  font:
+    normal:
+      family: "MesloLGS NF"
+      style: Regular
```
Comment (z0rc) [inline README.md:564]: Also this is default face to normal style, no point of specifying it here.

Comment (scribbles) [reply README.md:564]: It's based on the wiki entries below, which uncomment different values. I did not find a functional difference in testing.  Are you saying "font: normal: family: "MesloLGS NF"" is all that's required and the rest is implied?

[Alacritty wiki](https://github.com/alacritty/alacritty/wiki/Changing-the-default-font)
[Arch wiki](https://wiki.archlinux.org/index.php/Alacritty#Font)
[Gentoo wiki](https://wiki.gentoo.org/wiki/Alacritty#Font_configuration)

Comment (z0rc) [reply README.md:564]: > Are you saying "font: normal: family: "MesloLGS NF"" is all that's required and the rest is implied?

Yes, that's all that needed. Wiki's purpose is to be verbose to explain various use cases. But here specifying just font family is minimal configuration change to get expected result.

Comment (scribbles) [reply README.md]: Great, thanks. 001da9b includes this change.

**Comment by romkatv** [comment]
Thanks, @scribbles and @z0rc !

**Pull request merged**